grants/milestone_deliveries/season_2/package_manager/data/videos/README.md

```diff
@@ -0,0 +1,10 @@
+# Videos
+
+These research videos compliment the written content. They are saved to both YouTube and Filecoin.
+
+## YouTube
```

Comment:
These videos are amazing! Love the timestamps and everything. 

Super great to set an example for anyone else doing videos in the future.

Reply:
Thank you! I find that some people don't like watching videos, but others hate reading technical descriptions. Ultimately, I find both together are the best way to communicate technical ideas.